.github/workflows/codeql.yml

```diff
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.txt'
+      - '**/*.yaml'
+      - '**/*.json'
+      - '**/node_modules'
+      - '**/*.test.js'
+      - '**/*.test.ts'
+  pull_request:
+    branches: [ "master" ]
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.txt'
+      - '**/*.yaml'
+      - '**/*.json'
+      - '**/node_modules'
+      - '**/*.test.js'
+      - '**/*.test.ts'
+
+jobs:
+  analyze:
+    name: Analyze
```

Comment:
Like the workflow name we prefer to keep the id of the job and avoid setting a custom name. Please remove this line.